modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="cco-ccoctl-deleting-sts-resources_{context}"]
+= Deleting AWS resources with the Cloud Credential Operator utility
+
+After installing an {product-title} cluster with the Cloud Credential Operator (CCO) in manual mode with STS, you can use the CCO utility (`ccoctl`) to remove the AWS resources that `ccoctl` created. These resources are required only during changes that require elevated permissions, such as upgrades.
+
+This procedure can also be used to clean up resources after uninstalling a cluster that is configured to use the CCO in manual mode with STS.
```

Comment:
```suggestion
This procedure is meant to be used to clean up resources previously created by ccoctl after uninstalling a cluster that is configured to use the CCO in manual mode with STS.
```
Some background on how this works. If some of the resources ccoctl can create were created manually in a way that the names were customized before or after creation (ccoctl allows you to mix and match steps of desired/required) the tool will not be able to locate and delete the resources that don't follow ccoctl's naming/tagging scheme. These resources would need to be located and cleaned up manually. To;Dr the delete support assumes ccoctl created everything.

Reply:
Ok, cool - thanks for that :) I think I was a bit caught up between local and AWS-side items, and then also making this sort of in-line with other CCO topics, but this works inherently differently in that respect :+1: